src/Build.UnitTests/BackEnd/SdkResolverService_Tests.cs

```diff
@@ -196,6 +201,7 @@ public void AssertFirstResolverErrorsSupressedWhenResolved()
             var result = SdkResolverService.Instance.ResolveSdk(BuildEventContext.InvalidSubmissionId, sdk, _loggingContext, new MockElementLocation("file"), "sln", "projectPath", interactive: false, isRunningInVisualStudio: false, failOnUnresolvedSdk: true);
 
             result.Path.ShouldBe("resolverpath2");
+            _logger.BuildMessageEvents.Any(i => i.Message.StartsWith($"The SDK \"{sdk}\" was successfully resolved")).ShouldBeTrue();
```

Comment:
I was thinking about those verifications of logger and thought maybe we can simply introduce the test that verifies that the message is actually logged in success path and not otherwise , to keep the tests in mode "signle responsiblity" depending on what test actually checks, what do you think?

Reply:
Thanks! Done.